docs/en/success-guide/index.md

```diff
@@ -3,12 +3,12 @@ layout: page
 title: Setup and Success Guide
 advertise: Setup and Success Guide
 lang: en
-nav_link: Setup Guide Contents
+nav_link: Setup and Success Guide
 nav_level: 1
 nav_order: 100
 ---
 
-Version 1.1, released 2017-03-06
+Version 2.1.1, released 2017-06-09
```

Comment:
Don't like my idea of moving this to the "About" page, eh?  Sigh, I can live with it :).